File: 0268-missing-number/solution.java
class Solution {
    public int missingNumber(int[] nums) {
        Arrays.sort(nums);
        int n =nums.length;
        for(int i=0;i<n;i++){
            if(nums[i]==i){
                continue;
            }
            else{return i;}
        }
    return n;
    }
}

File: 0268-missing-number/solution.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        l=set([i for i in range(len(nums)+1)])
        z=l-set(nums)
        return list(z)[0]
        
